pyproject.toml

```diff
@@ -35,7 +35,7 @@ poetry = "^1.7.1"
 poetry-plugin-export = "^1.6.0"
 functions-framework = "^3.5.0"
 cron-validator = "^1.0.8"
-prediction-market-agent-tooling = { version = "^0.27.0", extras = ["langchain", "google"] }
+prediction-market-agent-tooling = { git = "https://github.com/gnosis/prediction-market-agent-tooling.git", branch = "evan/get_positions_liquid_only", extras = ["langchain", "google"] }
```

Comment:
Will update to 0.28.0 after https://github.com/gnosis/prediction-market-agent-tooling/pull/244 is merged